Merge changes from finalized 19.6 release

mokagio opened the pull request:
Notice there are no changes to the translations, because we shipped a beta just yesterday: #18372.

This is confirmed by my Fastlane output locally

```
[05:08:53]: ------------------------
[05:08:53]: --- Step: git_commit ---
[05:08:53]: ------------------------
[05:08:53]: $ git status /Users/gio/Developer/a8c/wpios/fastlane/metadata/\*\*/\*.txt --porcelain
[05:08:53]: Nothing to commit, working tree clean ✅.

...

[05:09:03]: ------------------------
[05:09:03]: --- Step: git_commit ---
[05:09:03]: ------------------------
[05:09:03]: $ git status /Users/gio/Developer/a8c/wpios/fastlane/jetpack_metadata/\*\*/\*.txt --porcelain
[05:09:03]: Nothing to commit, working tree clean ✅.
```

Unfortunately, I noticed this only later in the process, and the `**.lproj/Localizable.strings` commit attempt with nothing to commit got deleted in my not-infinite terminal scrollback.

hassaanelgarem committed:
Bump version number

oguzkocer commented:
@mokagio You'll need to abort the cherry-pick as otherwise `git` will continue as if you're still working on cherry picking. You can do this by `git cherry-pick --abort` or continue with the remainder of the cherry-pick command with `git cherry-pick --continue` after you resolve conflicts. When you abort the cherry-pick, it'll automatically go back to your previous commit, so you won't need to reset. Hope that helps!

mokagio committed:
Manually add missing release notes localizations for WP 19.6

All those localizations were still missing from GlotPress at the time of
this commit. Luckily, the Jetpack release notes copy was the same as
WordPress' this time around, so we can use the same translations.

mokagio committed:
Revert WordPress `tr` `subtitle.txt` to version <= 30 characters

mokagio commented:
Thanks @oguzkocer ! `git cherry-pick --abort` I'll keep it in mind 🙇‍♂️
